src/budy/static/css/report.css

```
@@ -0,0 +1,569 @@
+    grid-row-gap: 6px;
+    grid-template-columns: 160px 1fr;
+}
+
+.report-dt {
+    color: #757575;
+    font-size: 14px;
+}
+
+.report-dd {
+    font-size: 14px;
+    word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)